host/src/components/Responses/ResponsesGraph.jsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { Typography, makeStyles } from '@material-ui/core';
+import { VictoryChart, VictoryAxis, VictoryBar, VictoryContainer } from 'victory';
+import CustomTick from './CustomTick';
+import CustomLabel from './CustomLabel';
+import CustomBar from './CustomBar';
+
+export default function ResponsesGraph ({ 
+  data, 
+  questionChoices, 
+  statePosition, 
+  graphClickInfo, 
+  setGraphClickInfo 
+}) {
+  const classes = useStyles();
+  const [boundingRect, setBoundingRect] = useState({ width: 0, height: 0 });
+  const graphRef = useRef(null);
```

Comment:
Victory is the library we're using to do charting here and in ConfidenceResponseChart (in a later PR) as well as in upcoming features. While it's allowed us to generate the charts using D3 and to Figma design, there is a lot of specific formatting that is required throughout the component. To try and improve readability as much as possible, we've chosen to hoist all these values up to the top of the Charting component and then pass them down. Additionally, when we transition to the upgrade host design, we'll implement mui v5 with themeing and can hold a lot of these values (like the padding) in the theme itself and then share them between all the Victory components.